lib/live_debugger_web/live_components/filters_form.ex

```diff
@@ -21,7 +21,8 @@ defmodule LiveDebuggerWeb.LiveComponents.FiltersForm do
 
   @impl true
   def render(assigns) do
-    assigns = assign(assigns, :selected_filters_number, calculate_selected_filters(assigns.form))
+    assigns =
+      assign(assigns, :selected_filters_number, calculate_selected_filters(assigns.form) - 2)
```

Comment:
Not sure how this calculation should work after these changes - we'll have to wait for the designs. However, for this moment I'd assume that it should calculate only marked checkboxes. To achieve that I'd change `calculate_selected_filters` function - it should probably filter out fields that are not callback checkboxes